react/NestedSelect/NestedSelect.jsx

```diff
@@ -11,8 +12,10 @@ import ListItemText from '../ListItemText'
 import Divider from '../MuiCozyTheme/Divider'
 import ListItemIcon from '../MuiCozyTheme/ListItemIcon'
 import useBreakpoints from '../hooks/useBreakpoints'
+import Input from '../Input'
 
 import RightIcon from 'cozy-ui/transpiled/react/Icons/Right'
+import Typography from '@material-ui/core/Typography'
```

Comment:
You should import `Typography` from cozy-ui.

Reply:
> You should import `Typography` from cozy-ui.

Done

Reply:
Where?